### PR TITLE
docs: Documentation updates

### PR DESCRIPTION
**Pull request by aviv926**
This PR includes small updates:

* Add an updated screenshot of the works including duplicate detection
* Add information about reverse proxy
* Add a link to an official video on setting up Tailscale by Tailscale
* Deleting "duplicate" information
* Add a link to the Cursed knowledge page on troubleshooting.md (This page is not mentioned anywhere except in a release note.)
* Add community projects

### CODE REVIEW

File: docs/docs/guides/remote-machine-learning.md [hunk 1]
```diff
@@ -11,6 +11,10 @@ To alleviate [performance issues on low-memory systems](/docs/FAQ.mdx#why-is-imm
 Smart Search and Face Detection will use this feature, but Facial Recognition is handled in the server.
 :::
 
+:::danger
+When using remote machine learning, the thumb images are sent to the remote machine learning, use this option carefully when running this on a public computer or a paid processing cloud.
```
Comment (danieldietzler) [inline docs/docs/guides/remote-machine-learning.md:15]:
```suggestion
When using remote machine learning, the thumbnails are sent to the remote machine learning container. Use this option carefully when running this on a public computer or a paid processing cloud.
```
Generally I don't really like the sound of this and I'm not even sure we need it

File: docs/src/components/community-projects.tsx [hunk 2]
```diff
@@ -63,6 +63,16 @@ const projects: CommunityProjectProps[] = [
     description: 'Powershell Module for the Immich API',
     url: 'https://github.com/hanpq/PSImmich',
   },
+  {
+    title: 'Support mTLS with Home Assistant',
+    description: 'Allows access to Immich from public IP that did mTLS with Home Assistant.',
```
Comment (danieldietzler) [inline docs/src/components/community-projects.tsx:68]: Where is this related to HA?

File: docs/src/components/community-projects.tsx [hunk 3]
```diff
@@ -63,6 +63,16 @@ const projects: CommunityProjectProps[] = [
...
+    url: 'https://github.com/Tomasinjo/gatekeeper',
+  },
+  {
+    title: 'lrc-immich-plugin',
+    description: 'Export images from Lightroom Classic directly to Immich.',
+    url: 'https://github.com/Tomasinjo/gatekeeper',
```
Comment (danieldietzler) [inline docs/src/components/community-projects.tsx:74]: Copy paste issue

**Review by danieldietzler — COMMENTED** [review]
On another note, I think it'd be cool if the docs would get more my.immich.app links. I feel like that's much more helpful than some detailed guide with screenshots showing you how to get to that setting

**Comment by aviv926** [comment]
> On another note, I think it'd be cool if the docs would get more my.immich.app links. I feel like that's much more helpful than some detailed guide with screenshots showing you how to get to that setting

Totally agree! 
In the future, will we be able to support the possibility of redirection in case there are changes in the layout? 
Otherwise it would be tougher to update all the links than to add a record in the reference file.

Like we do with  '_redirects' file.

File: docs/docs/developer/troubleshooting.md [hunk 4]
```diff
@@ -4,6 +4,10 @@
 A great option to get assistance with troubleshooting is to join our [Discord](https://discord.immich.app) server, where we have a dedicated channel for `#contributing`.
 :::
 
+:::info Cursed knowledge
+[Cursed knowledge](/cursed-knowledge) we have learned as a result of building Immich that we wish we never knew.
+:::
+
```
[the remark this answers is not in the document]
Comment (aviv926) [reply docs/docs/developer/troubleshooting.md:10]: There is no other way to know about the existence of this page except in the release notes from a few weeks ago

File: docs/docs/guides/remote-access.md [hunk 5]
```diff
@@ -57,3 +61,4 @@ A remote reverse proxy like [Cloudflare](https://www.cloudflare.com/learning/cdn
 
 - Complex configuration
 - Depending on your configuration, both the Immich web interface and API may be exposed to the internet. Immich is under very active development and the existence of severe security vulnerabilities cannot be ruled out.
+- Reverse proxy [can weaken the privacy](https://www.youtube.com/results?search_query=cloudflare+privacy) of your information.
```
[the remark this answers is not in the document]
Comment (aviv926) [reply docs/docs/guides/remote-access.md:64]: Yes, I agree with you, but maybe because this is such an easy option to implement through cloudflare, it might be worth informing users that this is not necessarily a way that guarantees privacy, since cloudflare can view information as unencrypted if they want to.

File: docs/docs/guides/remote-machine-learning.md (hunk 1 above)
Comment (aviv926) [reply docs/docs/guides/remote-machine-learning.md:15]: I'll remove it if it's not relevant, I do think it's worth letting users know that there is a possibility of sensitive information being leaked if they choose to use a company that provides cloud processing power

File: docs/src/components/community-projects.tsx (hunk 2 above)
Comment (aviv926) [reply docs/src/components/community-projects.tsx:68]: https://www.reddit.com/r/immich/comments/1e8ebbl/comment/le7v7ip/

File: docs/docs/developer/troubleshooting.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (danieldietzler) [reply docs/docs/developer/troubleshooting.md:10]: Screenshot 2024-08-18 at 13 56 57
In the footer of the docs

File: docs/docs/guides/remote-access.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (danieldietzler) [reply docs/docs/guides/remote-access.md:64]: I get that, but I think I would prefer putting the focus on Cloudflare (and similar products) then and don't generalize it this much

File: docs/docs/guides/remote-machine-learning.md (hunk 1 above)
Comment (danieldietzler) [reply docs/docs/guides/remote-machine-learning.md]: But that's like internet 101. Of course it's not a good idea to send images around the world without any encryption. I think I'd be happy with only keeping the first part: "When using remote machine learning, the thumbnails are sent to the remote machine learning container."

File: docs/src/components/community-projects.tsx (hunk 2 above)
Comment (danieldietzler) [reply docs/src/components/community-projects.tsx:73]: This is like *really* weird lol. So basically, the idea is that the home assistant app will phone home from time to time, so whenever it phones home again we whitelist that IP address, thus allowing the immich app to bypass the mTLS handshake because the IP is whitelisted.

Comment (danieldietzler) [reply docs/src/components/community-projects.tsx:73]: I really don't like the whole concept of this, and imo it doesn't add any real security benefit either. I'll let the others chime in as well though

**Review by jrasm91 — APPROVED** [review]
